src/main/java/com/springCommerce/commerce/service/UserService.java

```diff
@@ -0,0 +1,59 @@
+package com.springCommerce.commerce.service;
+
+import com.springCommerce.commerce.dto.CreateUserRequest;
+import com.springCommerce.commerce.dto.UpdateUserRequest;
+import com.springCommerce.commerce.dto.UserDto;
+import com.springCommerce.commerce.dto.UserDtoConverter;
+import com.springCommerce.commerce.exception.UserNotFoundException;
+import com.springCommerce.commerce.model.User;
+import com.springCommerce.commerce.repository.UserRepository;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class UserService {
+
+    private final UserRepository userRepository;
+    private final UserDtoConverter userDtoConverter;
+
+    public UserService(UserRepository userRepository, UserDtoConverter userDtoConverter) {
+        this.userRepository = userRepository;
+        this.userDtoConverter = userDtoConverter;
+    }
+
+    public List<UserDto> getAllUsers(){
+        return userRepository.findAll().stream().map(userDtoConverter::convert).collect(Collectors.toList());
```

Comment:
Are you using a code formatter? Because this is too long to read, you can somethinng like this.
```.java
return userRepository.findAll()
   .stream()
   .map(userDtoConverter::convert)
   .collect(Collectors.toList());
```

Reply:
Actually I'm not using any code formatter but i will try. Thank you.